core/src/main/java/org/apache/metamodel/util/ResourceUtils.java

```diff
@@ -18,11 +18,31 @@
  */
 package org.apache.metamodel.util;
 
+import java.net.URI;
+
+import org.apache.metamodel.factory.ResourceFactoryRegistryImpl;
+import org.apache.metamodel.factory.ResourceProperties;
+import org.apache.metamodel.factory.SimpleResourceProperties;
+import org.apache.metamodel.factory.UnsupportedResourcePropertiesException;
+
 /**
  * Static utility methods for handling {@link Resource}s.
  */
 public class ResourceUtils {
 
+    public static Resource toResource(URI uri) {
+        return toResource(new SimpleResourceProperties(uri));
+    }
+
+    public static Resource toResource(String uri) {
+        return toResource(new SimpleResourceProperties(uri));
+    }
+
+    public static Resource toResource(ResourceProperties resourceProperties)
+            throws UnsupportedResourcePropertiesException {
```

Comment:
Ah right, but that's IMO more an issue with the overloaded methods :)
Maybe it's a bit of a religious discussion, but I would refer to some good (IMO) stackoverflow answers on the topic here:

http://stackoverflow.com/questions/20871763/should-i-declare-unchecked-exceptions-in-the-throws-specification

http://stackoverflow.com/questions/25743574/java-best-practice-for-declaring-unchecked-exception